scripts/slack_notifications.py

```diff
@@ -44,6 +44,12 @@ def send_notification():
         failure = nightly_slack_messages[message_key]
         message = {"text": ":blobonfire: Nightly build failed %s" % failure}
 
+    if workflow == "candidate":
+        if message_key == "success":
+            message = {"text": ":rocket: Release Candidate was successful!"}
+        else:
+            message = {"text": ":blobonfire: Release Candidate failed"}
+
     payload = json.dumps(message)
 
     response = requests.post(webhook, payload)
```

Comment:
nit: Also not quite part of this PR, but it's a bit more idiomatic to write

`requests.post(webhook, json=payload)` instead of calling `json.dumps` manually. I'm pretty sure doing this should work with any dict-like object as the `json` parameter